doc/connectivity/networking/api/wifi.rst

```diff
@@ -24,6 +24,46 @@ Currently, two types of Wi-Fi drivers are supported:
 * Networking or socket offloaded drivers
 * Native L2 Ethernet drivers
 
+Wi-Fi Enterprise test: X.509 Certificate header generation
+##########################################################
```

Comment:
Sorry, I gave you wrong heading format. This should be as per https://docs.zephyrproject.org/latest/contribute/documentation/guidelines.html#headings 

```suggestion
**************************************************
```